pkgs/development/python-modules/mediafile/default.nix

```
@@ -0,0 +1,29 @@
+{ buildPythonPackage
+, enum34
+, fetchPypi
+, lib
+, pythonOlder
+, pythonPackages
+}:
+buildPythonPackage rec {
+  pname = "mediafile";
+  version = "0.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "14vqvv4n0d93hbz9xpazcy3k7bfz3lapicp0qamjm656bi77j6fs";
+  };
+
+  propagatedBuildInputs = [
+    pythonPackages.mutagen
+    pythonPackages.six
+    pythonPackages.nose
+  ] ++ lib.optional (pythonOlder "3.4") enum34;
```

you disabled the python27 scenario anyway (no <3.4 interpreter will ever be included)
```suggestion
  ];
```